docs/api/vulnerability-info.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Vulnerability Info, NuGet API
+description: The data that allows clients to check packages for known vulnerabilities.
+author: zivkan
+ms.author: zivkan
+ms.date: 06/16/2023
+ms.topic: reference
+---
+
+# Vulnerability information
+
+NuGet client, starting from version 6.7, can download known package vulnerability information to use in scenarios such as checking packages during restore operations.
+
+The API consists of at least two files, the [vulnerability index](#vulnerability-index) and one or more [vulnerability data files](#vulnerability-data).
+Known vulnerability data can be partitioned into multiple files, and the vulnerability index provides clients with information needed to cache files, and update the cache, efficiently.
+
+The resource used for building this URL is the `VulnerabilityInfo` resource found in the [service index](service-index.md).
+
+## Versioning
+
+The following `@type` values are used:
+
+@type value                       | Notes
+--------------------------------- | -----
+VulnerabilityInfo/6.7.0 | The initial release
+
+## Suggested partitioning strategy
+
+The files listed in the vulnerability index should ideally be optimized to maximize caching, and therefore minimize updates to large files.
+This will allow clients to minimize the frequency it needs to download updates.
+
+A suggested strategy for vulnerability data partitioning is to have two files, `base.json` and `updates.json`.
+The `base.json` file is updated periodically (for example once a month), and contains all known vulnerabilities at the time the file is regenerated.
+The `updates.json` file should any new advisories published since `base.json` was last regenerated.
+This will allow clients to download the large `base.json` infrequently, while the frequently changing `updates.json` file is always relatively small.
+
+NuGet client does not provide a way for one file to update or remove entries from another file, so if any modifications are needed to any advisory in `base.json`, then `base.json` can be regenerated and all entries in `updates.json` merged.
+
+## Vulnerability index
+
+The vulnerability index is a JSON array of objects with the following properties:
+
+Name     | Type     | Required | Notes
+-------- | ---------| -------- | -----
+@name    | string   | yes      | A short name for the file, used as a cache key.
+@id      | string   | yes      | The full (absolute) URL to a vulnerability data file.
+@updated | DateTime | yes      | An ISO 8601 string representing the date and time the file was last updated, ideally with the UTC timezone.
```

Comment:
To be consistent with other APIs, the `Type` should be `string` not `DateTime`. I like the specificity here but there is no DateTime type in JSON.

Example: `published` in https://review.learn.microsoft.com/en-us/nuget/api/registration-base-url-resource?branch=pr-en-us-3077#catalog-entry